cpp/src/IceStorm/Makefile.mk

```diff
@@ -43,7 +43,7 @@ icestormadmin_sources           := $(addprefix $(currentdir)/,Admin.cpp \
                                                              IceStormInternal.ice)
 
 icestormdb_targetdir            := $(bindir)
-icestormdb_dependencies         := IcePatch2 IceDB
```

Comment:
This was a bogus dependency. icestormdb doesn't use IcePatch2 or any of its helper functions.